sagemaker-pipelines/tabular/caching/sagemaker-pipelines-caching-steps.ipynb

```diff
@@ -0,0 +1,936 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    mse_threshold = ParameterFloat(name="MseThreshold", default_value=6.0)
</pre><p>Do we need these 2 parameters? Seems they are not used in any steps in this notebook.</p><pre class="ql-syntax" spellcheck="false">model_approval_status
mse_threshold
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>